tenable/io/v3/vm/tags/schema.py

```diff
@@ -0,0 +1,139 @@
+'''
+Tags API Endpoint Schemas
+'''
+from marshmallow import Schema, fields, validate
+from marshmallow.decorators import post_dump, pre_load, validates_schema
+from marshmallow.exceptions import ValidationError
+
+from tenable.io.v3.base.schema.explore.filters import ParseFilterSchema
+
+
+class TagCategorySchema(Schema):
+    '''
+    Schema for tags category methods
+    '''
+    name = fields.Str(
+        validate = validate.Length(1, 127),
+        required=True
+    )
+    description = fields.Str()
+
+
+class AssetTagSchema(Schema):
+    '''
+    Schema for asset tags methods
+    '''
+    action = fields.Str()
```

Comment:
Is there a master list of allowed values for **action** field ? If yes, then add validation to check if value belongs to master list.